### DISCUSSION TITLE
Measurement mrids for a given object type

### DISCUSSION THREAD

**Discussion by shpoudel** [GRIDAPPSD/gridappsd-forum#95] [Q&A] [answered]
Hello,

I would like to know when the platform changes the measurement mrid for the objects. 

I query the object measurements for a given object type and I am mapping those measurements mrid with the meter in GridLAB-D offline to mimic the sensor service simulator. When I am running the code this morning, it seems like the measurements mrids for objects are updated.

**Comment by poorva1209** [comment]
We change the measurement MRIDs whenever the model is updated. Only the MRID of the feeder model object is never changed. We do have some process of keeping the MRIDs persistent between model updates but MRIDs still can change if the objects are updated, new attributes are added or new objects are added. 

When did you make the list of the measurements MRIDs that you have? I would suggest not to keep it hardcoded. Or you can use a specific release of blazegraph if your app doesn't care about new updates in model. 

@tonya1 Can you provide more information here on persistent MRIDs and when did we last updated models?

**Comment by tonya1** [comment, marked as answer]
We updated the models in the blazegraph develop container 10 days ago.  Alexander is looking into an issue and I expect we will be updating again soon.  If you would like to use the previous version, you can update the blazegraph image: in your docker_compose.yml file to the latest released version gridappsd/blazegraph:v2020.11.0.

**Comment by shpoudel** [comment]
Thank you, @poorva1209 and @tonya1
